Vic2ToHoI4/Source/Configuration.h

```diff
@@ -83,7 +84,7 @@ class Configuration::Factory: commonItems::parser
 	std::unique_ptr<Configuration> importConfiguration(std::istream& theStream);
 
   private:
-	void setOutputName(const std::string& V2SaveFileName);
+	void setOutputName(const std::string& V2SaveFileName, const std::string& OutputCustomName);
```

Comment:
With this changing, ensure the output name tests are still named well and set up correctly.